openmoc/compatible/opencg_compatible.py

```diff
@@ -282,36 +283,81 @@ def get_compatible_opencg_surfaces(opencg_surface):
   # Correct for OpenMOC's syntax for Surfaces dividing Cells
   boundary = opencg_surface.boundary_type
 
-  if opencg_surface.type == 'z-squareprism':
+  # If Cell is outside the SquarePrism (positive halfspace), add "outside" 
+  # of Surface halfspaces. Since OpenMOC does not have a SquarePrism Surface, 
+  # individual Cells are created for the 8 Cells that make up the outer region 
+  # of a SquarePrism.
+  #                 |                    |
+  #           0     |        1           |    2
+  #           ______|____________________|______
+  #                 |     SquarePrism    |
+  #           7     |   (-)  halfspace   |    3
+  #           ______|____________________|______
+  #                 |                    |  
+  #           6     |        5           |    4
+  #                 |                    |
+
```

Comment:
I was thinking the ascii art would be well placed at line 461 in place of the comment currently on that line.